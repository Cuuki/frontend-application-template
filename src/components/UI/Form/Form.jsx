import React from 'react';
import PropTypes from 'prop-types';

const Form = () => null;

Form.propTypes = {};
Form.defaultProps = {};

export default Form;
