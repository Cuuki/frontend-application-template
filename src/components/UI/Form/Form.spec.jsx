import React from 'react';
import {render} from '@testing-library/react';
import Form from './Form';

describe('Form', () => {
  it('renders', () => {
    const {container} = render(<Form />);

    expect(container.firstChild).toBeTruthy();
  });
});
