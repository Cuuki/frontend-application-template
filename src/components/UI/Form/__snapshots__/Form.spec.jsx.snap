// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form renders with children 1`] = `
<form
  class="sc-bdfBwQ"
>
  <input
    type="text"
  />
  <input
    type="password"
  />
  <input
    type="email"
  />
</form>
`;
